godtools/App/Features/ToolSettings/Presentation/ToolSettings/ToolSettingsView.swift

```diff
@@ -9,15 +9,18 @@ import SwiftUI
 
 struct ToolSettingsView: View {
     
+    @ObservedObject var viewModel: BaseToolSettingsViewModel
+    
     private let contentInsets: EdgeInsets = EdgeInsets(top: 20, leading: 20, bottom: 0, trailing: 20)
     private let separatorLineSpacing: CGFloat = 25
+    private let primaryTextColor: Color = Color(.sRGB, red: 84 / 256, green: 84 / 256, blue: 84 / 256, opacity: 1)
```

Comment:
Noticed several colors defined directly in the views-- should these go in the ColorPalette, or is ColorPalette only for colors that get used throughout the app?